### PR TITLE
Feat/lesq 305/blog breadcrumbs relocation [LESQ-305]

### PR DESCRIPTION
**Pull request by njblore**
## Description
Ticket: [Move blog breadcrumb out of nav bar](https://www.notion.so/oaknationalacademy/Move-blog-breadcrumb-out-of-nav-bar-c4452ab6611c4093bb77bcabe76cc63b?pvs=4)  
   
Changes:
- Removed breadcrumbs from nav bar on blog/webinar listing and post pages
- Added breadcrumb into these pages directly above content
- Mobile view should be unchanged 
  
## How to test

1. Go to https://deploy-preview-1912--oak-web-application.netlify.thenational.academy/blog
2. See the breadcrumbs below the nav
3. Click on a blog post
4. You should see the breadcrumbs below the nav bar

## Screenshots

How it used to look:
Screenshot 2023-09-19 at 13 22 05
Screenshot 2023-09-19 at 13 22 12

How it should now look:
Screenshot 2023-09-19 at 13 21 34
Screenshot 2023-09-19 at 13 21 42

### CODE REVIEW

File: src/components/Posts/PostHeader/PostHeader.tsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ const PostHeader: FC<PostHeaderProps> = ({ post, page }) => {
   return (
     <>
       <Flex
-        $mt={[40, 72]}
+        $mt={[40, 12]}
```
Comment (kimon-satan) [inline src/components/Posts/PostHeader/PostHeader.tsx:26]: I realise that you didn't start this pattern but we should try get rid of components using margins for their placement on the page.

**Review by kimon-satan — APPROVED** [review]

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
@njblore nice job. A couple of small changes if possible:

- [x] The padding between the breadcrumbs and the line is looking quite tight as if it was outside of the box. See attached screenshot 
- [ ] Is is possible to remove the grey line on the pages that have the yellow header box (so all landing pages). I've missed that and I don't think we need it there. 
Screenshot 2023-09-19 at 14 46 08

File: src/components/Posts/PostHeader/PostHeader.tsx (hunk 1 above)
Comment (njblore) [reply src/components/Posts/PostHeader/PostHeader.tsx]: 100%! I have refactored this here as it didn't appear to be used anywhere else than the `PostSingleLayout` so hopefully no unexpected side effects 👁️ 👁️

**Review by Monika-Koziol — APPROVED** [review]

**Pull request merged**